packages/appeals-service-api/src/lib/token.js

```diff
@@ -1,10 +1,12 @@
+const crypto = require('crypto');
+const TOKEN_LENGTH = 5;
+const BUFFER_LENGTH = 20;
 const createToken = () => {
-	const token = [];
-	for (let i = 0; i < 5; i += 1) {
-		const num = Math.floor(Math.random() * 9 + 1);
-		token.push(num);
-	}
-	return token.join('');
+	return crypto
+		.randomBytes(BUFFER_LENGTH)
```

Comment:
Nice. I like the approach and I can see you've allowed for ~20 more characters than we need, presumably to account for the fact that  we might get a few `+`s or `/`s amongst the characters in the base64 encoded string. I'm wondering whether we need a length check on the string before we return it though. Much as it's highly unlikely, there is a chance we could get over 20 non alpha-numeric characters so theoretically could end up with a string containing less than 5 characters.

Reply:
How should it work in the failure case? Should it try again with the same algorithim or just return a 5 digit random number as a failover?

Reply:
I think that scenario would be sufficiently rare that you could try again and be pretty confident of a 5 digit code the next time.